app/src/main/java/com/codesoom/assignment/dto/UserLoginData.java

```diff
@@ -0,0 +1,26 @@
+package com.codesoom.assignment.dto;
+
+import com.github.dozermapper.core.Mapping;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+
+@Getter
+@Builder
```

Comment:
`@Builder`를 클래스 시그니처 위에 달아주지 마시고, 생성자를 만든 다음 생성자 위에 달아 주세요. (이렇게 하는 이유도 조사해 보세요)

Reply:
생성자를 만든 다음 생성자 위에 달면 생성자에 포함된 필드만 빌더에 포함됩니다.
>Finally, applying @Builder to a class is as if you added @AllArgsConstructor(access = AccessLevel.PACKAGE) to the class and applied the @Builder annotation to this all-args-constructor. This only works if you haven't written any explicit constructors yourself. If you do have an explicit constructor, put the @Builder annotation on the constructor instead of on the class. Note that if you put both `@Value` and `@Builder` on a class, the package-private constructor that `@Builder` wants to generate 'wins' and suppresses the constructor that `@Value` wants to make.
- [`@Builder`](https://projectlombok.org/features/Builder)

- `@Builder`를 class위에 선언하면 `@AllArgsConstrutor`를 붙인 것과 같이 되는데 필수 매개 변수가 되기 때문에 필요한 생성자 위에 달아주는 것이 적절한 책임을 주고 안전하게 객체를 생성하기 때문일까요?

Reply:
그렇습니다.